dev-aws/kafka-shared/iam/iam.tf

```diff
@@ -233,3 +233,28 @@ resource "kafka_topic" "iam_revoked_v1" {
     "cleanup.policy"    = "delete"
   }
 }
+
+
+resource "kafka_topic" "iam_credentials_v1_public" {
```

Comment:
I'm thinking if we can create this topic directly in the MSK cluster, but I see the app is producing to this one, but consuming from kafka_topic.iam_credentials_v1 which would be possible, but it needs different brokers in the deployment